packages/airnode-admin/src/cli.ts

```diff
@@ -104,6 +104,32 @@ const COMMON_COMMAND_ARGUMENTS = {
     demandOption: true,
     describe: 'Indefinite whitelist status that the requester will have',
   },
+  transactionOverrides: {
+    'gas-limit': {
+      type: 'string',
+      describe: 'Transaction gas limit override',
+    },
+    'gas-price': {
+      type: 'string',
+      describe: 'Transaction gas price override (in gwei)',
+    },
+    'max-fee': {
+      type: 'string',
+      describe: 'Transaction maximum fee (in gwei)',
+    },
+    'max-priority-fee': {
+      type: 'string',
+      describe: 'Transaction maximum priority fee (in gwei)',
+    },
+    value: {
```

Comment:
I'm not sure about adding `value` as an argument to all (or any) commands as it is only needed in `Payable` transactions and it seems it's only used for airnode's `fulfillWithdrawal` method which is not included in the CLI as a command.

Reply:
Yeah, I see no reason in adding it there. Maybe we could also remove it from `fulfillWithdrawal` ? Since that value is not needed for any other command.